README.md

```diff
@@ -72,12 +70,9 @@ sources, and line numbers of addresses involved in a process.
   }
 ```
 
-`sym_srcs` is a list of symbol sources in a process.
-However, there is only one `SymbolSrcCfg::Process {}` here.
-`SymbolSrcCfg::Process {}` is a convenient variant for loading all objects,
-i.e., binaries and shared libraries, mapped in a process.  Therefore, developers
-do not have to specify each object and its base address with
-`SymbolSrcCfg::Process {}`.
+`cfg` is the configuration to use when symbolizing addresses. The
+`SymbolSrcCfg::Process` variant used here instructs the symbolizer to use
```

Comment:
up for discussion and follow up: "SymbolSrcCfg" reads a bit like a soup of letters. Given we pass it to symbolize API, should we simplify this to "SymbolizeCfg" or "SymbolizeCtx" or "SymbolizeOpts" or something along those lines?

Reply:
Yes we should change that. I haven't overhauled naming in the existing API yet, because it will see various changes. But this will get changed in the Rust and C side eventually.